views/funding/create.php

```diff
@@ -17,22 +18,38 @@
 Select2BootstrapAsset::register($this);
 ?>
 
-<?php ModalDialog::begin(['header' => Yii::t('XcoinModule.base', 'Select wanted asset'), 'closable' => false]) ?>
+<?php ModalDialog::begin(['header' => Yii::t('XcoinModule.base', 'Select wanted asset and define exchange rate'), 'closable' => false]) ?>
```

Comment:
To accomodate the new translation format coming in the next module version, please change all `XcoinModule.base` occurences in your modified files (only files commited in this pr) to `XcoinModule.${view_name}` considering the `view_name` as in `views/${view_name}/`, like in this line `XcoinModule.base` -> `XcoinModule.funding`